tests/system/test_system_async.py

```diff
@@ -0,0 +1,1000 @@
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import datetime
+import math
+import pytest
+import operator
+import os
+import re
+
+from google.oauth2 import service_account
+
+from google.api_core.exceptions import AlreadyExists
+from google.api_core.exceptions import FailedPrecondition
+from google.api_core.exceptions import InvalidArgument
+from google.api_core.exceptions import NotFound
+from google.cloud._helpers import _datetime_to_pb_timestamp
+from google.cloud._helpers import UTC
+from google.cloud import firestore_v1 as firestore
+from test_utils.system import unique_resource_id
+
+FIRESTORE_CREDS = os.environ.get("FIRESTORE_APPLICATION_CREDENTIALS")
```

Comment:
Most projects use the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to locate the service account to use.

@crwilcox Is this accurate for python projects?

Reply:
@BenWhitehead  yep, specifically for firestore. this way we can use the same kokoro config :) The project that is `GOOGLE_APPLICATION_CREDENTIALS` is datastore

Reply:
From what I remember, Firestore uses a different environment variable than other Python projects. This is consistent with the sync version of Firestore.